tests/book.test.js

```diff
@@ -0,0 +1,209 @@
+const { expect } = require('chai');
```

Comment:
Indentation in this file is pretty all over the place, sometimes 2, sometimes 4 and once way more than that! It would be worth spending a bit of time sorting out your linter settings, as it would take care of all of this for you